### ISSUE TITLE
Add more aria-labels into search results

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1753]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

search results page (both grouped and "all")

### Description of error / warning

On an individual search result, there is a lot of content that's not labeled. there are cues for visual users but there isn't enough context when you're navigating with VoiceOver.

- the breadcrumbs in each result are supposed to give context about what sort of collection the material is in, but it just reads as a list of links (the page breacrumbs say "navigation breadcrumb" but I'm not sure if that's appropriate here)
- the highlighted snippets aren't labeled / announced as such, they are just read as text
- the description isn't announced as such, so it's not clear when you've moved from hit highlighting to resource description

When I'm navigating this way I do wish that the description would be read first. I feel like my first question is "what is this?" and then my second question might be "why did I get it?"

### WCAG success criterion

[4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG22/#name-role-value)

"ARIA is used appropriately to enhance accessibility when HTML is not sufficient."

### WCAG conformance level

A

### Recommended fix

Consider putting the description first.

Add more aria-labels for the different data areas of the search result card.

### Tool used to find the problem

VoiceOver

### Screenshot

<img width="942" height="435" alt="Image" src="https://github.com/user-attachments/assets/68218248-605f-4c52-a0c1-61b869bd0813" />
